dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/BootstrapConfig.java

```diff
@@ -133,6 +133,14 @@ public File run() throws Exception {
 
     }
 
+    protected void findLocations(String newServerName,
```

Comment:
Lost the javadoc on this method.  Although I question the need for javadoc on classes we do not actually generate javadoc for.

Reply:
One comment of the overloaded method signature should suffice here.